presto-spi/src/main/java/io/prestosql/spi/security/GroupProvider.java

```diff
@@ -17,5 +17,14 @@
 
 public interface GroupProvider
 {
+    /**
+     * Gets all groups for a specific user.
+     * <p>
+     * Groups are used by Presto to control the access to a
+     * resource.
+     *
+     * @param user the representation of user executing a query in Presto
```

Comment:
```suggestion
     * @param user the representation of an user executing a query in Presto
```

Reply:
It is "a user" not "an user" https://english.stackexchange.com/a/105117